__tests__/app.test.js

```diff
@@ -29,23 +29,23 @@ describe("nc news", () => {
         });
     });
   });
-describe("Get /api/articles/", () => {
-  test("Responds with 200 and array of article objects ", () => {
-    return request(app)
-      .get("/api/articles")
-      .expect(200)
-      .then(({ body: { articles } }) => {
-        articles.forEach((article) => {
-          expect(article).toHaveProperty("author");
-          expect(article).toHaveProperty("title");
-          expect(article).toHaveProperty("article_id");
-          expect(article).toHaveProperty("body");
-          expect(article).toHaveProperty("topic");
-          expect(article).toHaveProperty("created_at");
-          expect(article).toHaveProperty("votes");
-          expect(articles.length).toBe(12);
+  describe("Get /api/articles/", () => {
+    test("Responds with 200 and array of article objects ", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          articles.forEach((article) => {
+            expect(article).toHaveProperty("author");
+            expect(article).toHaveProperty("title");
+            expect(article).toHaveProperty("article_id");
+            expect(article).toHaveProperty("body");
+            expect(article).toHaveProperty("topic");
+            expect(article).toHaveProperty("created_at");
+            expect(article).toHaveProperty("votes");
+            expect(articles.length).toBe(12);
```

Comment:
Probably would move this length check outside of the forEach, waste of resources to check the length each time BUT this such a small bit of correction